TempMail/ContentView.swift

```diff
@@ -9,19 +9,22 @@
 import SwiftUI
 
 struct ContentView: View {
-    
-    var email_addr: String
+//    variable for Email struct
+    @ObservedObject var email = Email()
```

Comment:
Add the type identifier to the email object: @ObservedObject var email: Email = Email()